### PR TITLE
SC-13. Feature 5. 1 Edit contacts functionality

### LINKED ISSUE #9: SC-13. Feature 5.1 Edit Contacts functionality

### PR DESCRIPTION
**Pull request by DayOut**

### CODE REVIEW

File: src/handlers/contacts_handler.py [hunk 1]
```diff
@@ -26,25 +28,21 @@ def add_contact(args: list, book) -> str:
...
-                try:
-                    record.phones[index] = Phone(new)
-                    return "Contact updated."
-                except PhoneVerificationError as e:
-                    return e.message
+    if contact:
+        contact.edit_phone(old, new)
 
-        return f"Contact {name} does not have entered phone number."
+        return f"Number changed successfully."
```
Comment (yuratouch) [inline src/handlers/contacts_handler.py]: please change to "Phone number successfully updated."

File: src/modules/contact.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from src.modules.name import Name
-from src.modules.exceptions import PhoneVerificationError
+from src.modules.exceptions import (PhoneAlreadyExistsError, PhoneNotFoundError)
```
Comment (yuratouch) [inline src/modules/contact.py]: Please remove ().
let it be just PhoneAlreadyExistsError, PhoneNotFoundError

**Review by yuratouch — CHANGES_REQUESTED** [review]
edit contact functionality approved. 
new custom errors approved. 
Please do not include commits related to a previous task (SC-9) into this PR.

File: src/handlers/contacts_handler.py (hunk 1 above)
Comment (DayOut) [reply src/handlers/contacts_handler.py]: Done

File: src/modules/contact.py (hunk 2 above)
Comment (DayOut) [reply src/modules/contact.py]: Done

**Review by yuratouch — APPROVED** [review]

**Pull request merged**